mitodl/odl-video-service
# Rename a couple UWSGI environment variables, remove redundant if-env blocks

#### What are the relevant tickets?
Related to https://github.com/mitodl/micromasters/issues/4597

#### What's this PR do?
 - Rename `UWSGI_THREAD_COUNT` to `UWSGI_THREADS`, and `UWSGI_PROCESS_COUNT` to `UWSGI_PROCESSES`
 - Remove redundant if-not-env blocks

#### How should this be manually tested?
Rename the above environment variables if you have them set, then make sure the site loads on your local instance
